app/components/ExternalLinksBlock/components/ExternalLink/ExternalLink.jsx

```diff
@@ -30,6 +30,11 @@ export default function ExternalLink({ className, isExternalLink, isSocialLink,
     iconClasses = `fa ${icon} fa-fw`;
     linkLabel = path;
     linkProps['aria-label'] = socialIcons[site].ariaLabel;
+    if (site === 'wordpress') {
+      linkProps.href = `https://${path}.${site}`;
```

Comment:
I need to check that Wordpress URLs are constructed properly. If `site === 'wordpress'`, then I think the result might be `linkProps.href = 'https://shaun.wordpress'`

Reply:
Ah, `site` will be `wordpress.com`, so we need to change it to `if (site === 'wordpress.com')`

Reply:
Sorry! I meant `if (site === 'wordpress.com/')`